pkgs/applications/networking/instant-messengers/chatterino2/darwin-frameworks.patch

```diff
@@ -0,0 +1,27 @@
+Fix build on macOS.
```

Comment:
Slight nit: the comment about the patch should go above the patch in the derivation, not inside the patch itself. The first place people look (or at least, the first place *I* look) for the reasoning behind a patch is in the derivation itself.